com.exok.Celeste64.yml

```diff
@@ -0,0 +1,54 @@
+id: com.exok.Celeste64
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  arch:
+    x86_64:
+      env:
+        RUNTIME: "linux-x64"
+# support for aarm64 will be added in the future
+#    aarch64:
+#      env:
+#        RUNTIME: "linux-arm64"
+  prepend-pkg-config-path: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig"
+  append-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+command: celeste64
+finish-args:
+  - --device=all
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+
+modules:
+  - name: celeste64
+    buildsystem: simple
+    only-arches:
+      - "x86_64"
```

Comment:
You need this too https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on